test/suites/standard1/kustomize.robot

```diff
@@ -79,6 +96,12 @@ Setup Suite    # robocop: disable=too-long-keyword
     ${ns}=    Generate Manifests    ${ETC_SUBDIR}
     Set Suite Variable    \${ETC_SUBDIR_NAMESPACE}    ${ns}
 
+    # Used by "Delete from /etc/microshift/manifestsd"
+    ${rand}=    Generate Random String
+    Set Suite Variable    \${DELETE_ETC_SUBDIR}    /etc/microshift/manifests.d/${rand}
+    ${ns}=    Generate Manifests    ${DELETE_ETC_SUBDIR}
+    Set Suite Variable    \${DELETE_ETC_SUBDIR_NAMESPACE}    ${ns}
+
```

Comment:
Are we missing the dir deletion in the teardown, similar to
```
Remove Manifest Directory    ${ETC_SUBDIR}
```

Reply:
Ah, I see we're deleting it in the test. It's a bit assymetric with others.
Not sure how to improve this, but feel free to ignore if you think it's less important.

Reply:
PTAL if the way I addressed is acceptable

Reply:
I think it's much better, thank you. Let's resolve this comment.